assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx

```diff
@@ -964,7 +1054,12 @@ const ExternalActivity: React.FC<PartComponentProps<CapiIframeModel>> = (props)
         mFormatted[baseKey] = updatedVar;
         sendFormedResponse(simLife.handshake, {}, JanusCAPIRequestTypes.VALUE_CHANGE, mFormatted);
       }
-      sendFormedResponse(simLife.handshake, {}, JanusCAPIRequestTypes.VALUE_CHANGE, formatted);
+      if (
```

Comment:
Just making sure that, we don't send 'bindTo' variables in init state as we already send them after we indicate the CAPI TYPE = 14 i.e. initial setup complete